tests/test_playwright.py

```diff
@@ -190,6 +190,22 @@ def test_base_url(page, browser_name):
     result.assert_outcomes(passed=1)
 
 
+def test_slowmo(testdir: Any) -> None:
+    testdir.makepyfile(
+        """
+        from time import monotonic
+        def test_slowmo(page):
+            start_time = monotonic()
+            page.goto("https://github.com")
+            page.type("input[name=user_email]","test@test.com")
+            end_time = monotonic()
+            assert end_time - start_time >= 13
```

Comment:
How do you come to the 13? Seems like it could end up in flaky tests.

Reply:
Since every letter takes one second i.e. `test@test.com` is of 13 letters so it will take at least 13 sec not falky @mxschmitt

Reply:
Gotcha, lets store the email in a temporary variable and use len(input_value) e.g. so its more clear.

Reply:
@mxschmitt I have made changes, review again